packages/ketcher-core/src/application/render/draw.ts

```diff
@@ -81,43 +81,112 @@ function arrow(
   item: ArrowItem,
   length: number,
   angle: number,
-  options: RenderOptions
+  options: RenderOptions,
+  isResizing: boolean
 ) {
+  const shouldApplySnappingStyle =
+    isResizing &&
+    ['0', '-0', '90', '-90', '180', '-180'].includes(angle.toFixed())
```

Comment:
'-0' - not sure that js knows about such cases. Suggest removing this item from the array.